docs/Overview.rst

```diff
@@ -56,3 +56,12 @@ The validation techniques used are listed here below and expanded in :ref:`valid
 
 * Comparison to other model
 
+
+Access MVS Server API
+---------------------
+
+The multi-vector simulator can be used to simulate energy systems directly via the browser using the API of the server which processes the simulation. Users can interact with the API at this `URL <https://mvs-eland.rl-institut.de/>`_.
```

Comment:
```suggestion
The Multi-Vector Simulator can be used to simulate energy systems via the browser using the `API on the RLI server <https://mvs-eland.rl-institut.de/>`__. It processes the parsed JSON input file, and returns the JSON result file.
```
Somehow there need to be two underscores, otherwise there is a RTD warning.